apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/AIPolicyEditorPanel.utils.ts

```diff
@@ -74,3 +74,13 @@ COMMIT;
 `.trim()
   }
 }
+
+export const generatePolicyDefinition = (policy: PostgresPolicy) => {
+  return `
+CREATE POLICY "${policy.name}" on "${policy.schema}"."${policy.table}"
+AS ${policy.action} FOR ${policy.command}
+TO ${policy.roles.join(', ')}
+USING (${policy.definition})${policy.check === null ? ';' : ''}
```

Comment:
Do policies always have a using definition? For example `INSERT` expressions only have `WITH CHECK` and no `USING`

Reply:
ahhh good point - lemme update